### DISCUSSION TITLE
6.0.0-alpha-1

### DISCUSSION THREAD

**Discussion by dustinmoris** [giraffe-fsharp/Giraffe#498] [General]
Upgraded to .NET 6 and F#'s new native `task` computation expression.

<hr /><em>This discussion was created from the release <a href='https://github.com/giraffe-fsharp/Giraffe/releases/tag/v6.0.0-alpha-1'>6.0.0-alpha-1</a>.</em>

**Comment by xdaDaveShaw** [comment]
Just an FYI - I've updated everything to .net6.0 and this alpha and everything seems to work OK.

I removed my `ply` reference and updated my `open`s from `FSharp.Control.Tasks` to `FSharp.Control`

Nice one :)
